app/models/concerns/hyrax/solr_document/metadata.rb

```diff
@@ -66,6 +66,7 @@ def self.coerce(input)
         attribute :source, Solr::Array, solr_name('source')
         attribute :date_created, Solr::Array, solr_name('date_created')
         attribute :rights_statement, Solr::Array, solr_name('rights_statement')
+        attribute :rights_notes, Solr::Array, solr_name('rights_notes')
```

Comment:
@blancoj I see that we store this as a Solr::Array. How does that relate or not with `multiple: false`?